modules/examples/sources/msg-generator/msg-generator-source-options.h

```diff
@@ -73,5 +77,9 @@ msg_generator_source_options_set_template(MsgGeneratorSourceOptions *self, LogTe
   log_template_unref(self->template);
   self->template = log_template_ref(template);
 }
-
+static inline void
+create_log_name_value_pair(MsgGeneratorSourceOptions *self, const gchar *name, const gchar *value)
```

Comment:
I believe this should not be inline, and should be made public in the header file, so `msg-generator-grammar.ym` can reach it.